Makefile.am

```diff
@@ -23,3 +23,7 @@ ACLOCAL_AMFLAGS = -Im4/autotools
 SUBDIRS = m4 include lib src doc man
 dist_doc_DATA = README.md
 dist_noinst_DATA = tlog.spec
+# Remove *.gcno files from all directories.
```

Comment:
This comment doesn't match what's being done, and is not really helpful. It's better to explain what those files are and where they come from. Please fix.